platforms/documentation/docs/src/snippets/buildlifecycle/basic/groovy/build.gradle

```diff
@@ -11,7 +11,9 @@ tasks.register('test') {
 }
 
 tasks.register('testBoth') {
-	doFirst {
+    mustRunAfter("test")
```

Comment:
🤔 Can we make it so this doesn't show up in the snippet in the docs? I suppose this should be possible, right? Same for the other example.

Reply:
I know how to do this ;) You need to use tags like e.g. here:
https://github.com/gradle/gradle/blob/24ef4fc04a5c89c8a0faa3588ab78c2d14f684c7/platforms/documentation/docs/src/docs/userguide/authoring-builds/structuring/composite_builds.adoc?plain=1#L67-L70
https://github.com/gradle/gradle/blob/c42798b4690cbec3294ff3d927c296e88749fa3b/platforms/documentation/docs/src/samples/build-organization/composite-builds/basic/groovy/build.gradle#L1-L13

Reply:
@wolfs has the solution. I added an "out-of-order" flag in my fix: https://github.com/gradle/gradle/pull/29128